modules/imgproc/src/resize.cpp

```diff
@@ -1261,21 +1261,16 @@ struct VResizeLinearVec_32s8u
         v_int16 b0 = vx_setall_s16(beta[0]), b1 = vx_setall_s16(beta[1]);
 
         if( (((size_t)S0|(size_t)S1)&(CV_SIMD_WIDTH - 1)) == 0 )
```

Comment:
Shouldn't `CV_SIMD_WIDTH` be replaced with `VTraits<v_uint8>::vlanes`? Or what is the meaning of this condition? Here and below.

Reply:
Yes, we should use `VTraits<v_uint8>::vlanes()` here and below. Fixed.

And I personally think that the meaning of this condition is to judge whether `S0` and `S1` (as addresses) are aligned with `CV_SIMD_WIDTH` (`CV_SIMD_WIDTH-1` acts as a mask here). If so, we can use` vx_load_aligned`, otherwise use `vx_load`. BTW, the implementation in the two load functions RVV is the same.